libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1360,6 +1361,15 @@ GCS_MAVLINK::update_receive(uint32_t max_time_us)
             hal.util->persistent_data.last_mavlink_msgid = msg.msgid;
             hal.util->perf_begin(_perf_packet);
             packetReceived(status, msg);
+            if (msg.sysid == sysid_my_gcs() && total_msg_cnt != 0) {
+                lost_msg_cnt += uint8_t(msg.seq - last_gcs_msg_seq - 1);
+                total_msg_cnt += uint8_t(msg.seq - last_gcs_msg_seq);
+                last_gcs_msg_seq = msg.seq;
+            } else if (total_msg_cnt == 0) {
```

Comment:
Maybe make this the first clause so you don't need to repeat the comparison against total_msg_count?

Reply:
agreed, would actually speed it up a tiny bit too